### DISCUSSION TITLE
v1.6.8

### DISCUSSION THREAD

**Discussion by deniscerri** [deniscerri/ytdlnis#313] [General]
### Notable Change

- Made app always show the quick download card and asynchoronously load data while you are in it. Quick Download now if its on, it wont load data at all inside the card. This is useful in case people dont want to wait for redundant data when they end up quickly downloading anyway. You can cancel the update process by tapping the card title. If you have automatic format updating enabled, the formats will also be updated while you are in the card with shimmer effects. After data loads it will automatically fill in the data and in case you have edited out the title or author it will suggest you to update the fields on the side instead of replacing them. In case if the data was a playlist all along the card will dismiss and show you the playlist selector. Happy downloading :)
----

## Other stuff:
- removed trim filenames from command type downloads
- fixed app not showing formats when it has generic formats
- fixed author getting written as NA for kick videos
- added separate channel for the worker notification that users can turn off
- include search history when searching
- removed scroll bug from command tab
- added spacing between command template title and ok button in selection card
- made download progress not dublicate in terminal
- made ability to store terminal state
- added ability to create multiple terminal instances and show them as a list similar to download queue
- fixed thumbnail download not working
- fixed app crashing when clicking on format updated notification
- fixed app crashing when double clicking format on multiple download card
- added custom sponsorblock api preference
- removed contextual app bar when you its enabled and the user taps the log button in the erorred tab
- added shimmer when loading data in the download card
- fixed app showing no formats if there were no common formats. Now it will give you generic formats
- made open command template list be half the screen, shortcuts third of the screen so the user can see what its being added
- fixed sometimes app slipping queued downloads even though its told to pause all
- fixed trim filenames cutting files too short
- made mediastore scanning of files one by one
- fixed filename template not working in multiple download card
- fixed -F in terminal not being inline
- added preferred audio codec
- made auto update on boot if there are no active downloads
- fixed format text overlapping
- added a new error dialog in cases yt-dlp data fetching in the home screen fails. You can copy the log
- formats auto updating as soon as the download card opens if auto-update is on
- added preferred audio format always in the video tab
- made app post downloads for queue in chunks
- made app always save logs in case it fails, and if succeeds and logs are off it deletes it
- fixed app navigating to home screen when cancelling download card in history screen
- added a button to skip incoming app update so it wont bother you anymore
- fixed settings not restoring some fields when coming from a backup
- fixed crunchyroll not working with cookies
- added search for command templates
- added sort filtering for command templates
- added all shortcuts inside filename template creation dialog. Long click them to see the explanation
- added preference to hide elements from the download card
- made avc1 and m4a as preferred codecs for noobs
- u can edit the duplicated download item right from the error dialog or access the history item to view the file
- added extractor args lang when searching in yt-dlp
- removed webarchive search engine since its not supported anymore
- fixed terminal prematurely closing
- made format auto-update on by default for new users
- fixed main activity getting removed from recents when using the app with rvx
- added ability to have the last used command template for the next download
- fixed app crashing in landscape logs
- fixed app constantly going back to home in landscape or config change. now it keeps state
- add subtitle language suggestions in the subtitle dialog
- made command template scroll state hold even if fragment is destroyed
- added slovak sk language
- fixed terminal icon being blank in white mode, and now its red
- fixed share files from notification showing 2 files even though its 1
- fixed history item not being deleted from the bottom sheet
- cleared outdated player urls for stale result items
- added export cookies as file
- added export command templates for selected templates
- added icons for history details sheet chips
- added markdown in the update dialog
- and other random bug fixes

## Autogenerated Changes
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/283
* Create README-id.MD (Indonesian README) by @teddysulaimanGL in https://github.com/deniscerri/ytdlnis/pull/291
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/288
* Update README-pt.md by @gigoloinc in https://github.com/deniscerri/ytdlnis/pull/293
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/292
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/301
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/305
* Translations update from Hosted Weblate by @weblate in https://github.com/deniscerri/ytdlnis/pull/310

## New Contributors
* @teddysulaimanGL made their first contribution in https://github.com/deniscerri/ytdlnis/pull/291
* @gigoloinc made their first contribution in https://github.com/deniscerri/ytdlnis/pull/293

**Full Changelog**: https://github.com/deniscerri/ytdlnis/compare/v1.6.7...v1.6.8

<hr /><em>This discussion was created from the release <a href='https://github.com/deniscerri/ytdlnis/releases/tag/v1.6.8'>v1.6.8</a>.</em>

**Comment by Texdiablo82** [comment]
Sorry for my ignorance but what version should I download for my honor Magic 5 pro? Thanks

**Reply by zaednasr** [reply]
arm64. armv7 is for old phones
